packages/front-end/pages/settings/webhooks/index.tsx

```diff
@@ -29,10 +30,9 @@ const WebhooksPage: FC = () => {
       <h1>SDK Endpoint Webhooks</h1>
       <Webhooks />
 
-      {/* TODO: Include this code when we are ready to launch Event-based Web Hooks */}
-      {/*<div className="mt-5">*/}
-      {/*  <EventWebHooksPage />*/}
-      {/*</div>*/}
+      <div className="mt-5">
+        <EventWebHooksPage />
```

Comment:
This seems to have some extra padding coming from the `container pagecontents` classes.

Reply:
Good eye! 👁️ I've removed the `container` class which added an extra 8px horizontally. Thanks!